src/styles/blocks/start-day.scss

```
@@ -0,0 +1,6 @@
+@each $day, $margin in $days {
+  .calendar--start-day--#{$day}
+  &:first-child {
+    margin-left: $margin * ($cell-size + $gap + 1.6 * $border-width);
```

```suggestion
    margin-left: $margin * ($cell-size + $gap + 2 * $border-width);
```
Consider fixing it to avoid element shifting